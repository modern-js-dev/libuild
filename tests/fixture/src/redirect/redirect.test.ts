import { getLibuilderTest } from '@modern-js/libuild-test-toolkit';

describe('fixture:redirect', () => {
  it('redirect', async () => {
    const bundler = await getLibuilderTest({
      root: __dirname,
      resolve: {
        alias: {
          '@': 'src',
        },
      },
      bundle: false,
      input: ['./src'],
    });
    await bundler.build();
    bundler.expectJSOutputMatchSnapshot();
  });
});
