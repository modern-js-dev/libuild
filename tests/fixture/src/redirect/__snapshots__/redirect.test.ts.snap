// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`fixture:redirect redirect 1`] = `
"import \\"./style.module.css\\";
var style_module_default = { \\"a\\": \\"a_ce8e5\\" };
export {
  style_module_default as default
};
"
`;

exports[`fixture:redirect redirect 2`] = `
"import b from \\"./assets/logo.5d5d9eef.svg\\";
import a from \\"./style.module\\";
export {
  a,
  b
};
"
`;
